scripts/acn/acn-deployment/ingress_tcp_routing/tcp_routing_config_map.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tcp-services
+  namespace: ingress
+data:
+  9003: "dht-node/acn-node-9003:9003"
+  9004: "dht-node/acn-node-9004:9004"
+  9005: "dht-node/acn-node-9003:11003"
+  9006: "dht-node/acn-node-9004:11004"
+---
```

Comment:
Note we have to open a specific port to listen for TCP traffic on each worker node in the cluster. Traffic is routed through aws & our route 53 setup however.